### DISCUSSION TITLE
skoala/intro/advantages/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1492] [Announcements]
# skoala/intro/advantages/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/intro/advantages/

**Comment by liyuerich** [comment]
微服务引擎支持传统和云音乐的注册中心

应该是 云原生 的注册中心吧？

**Reply by Michelle951** [reply]
fixed：
<img width="958" alt="image" src="https://github.com/DaoCloud/DaoCloud-docs/assets/35519562/0d47de0b-c470-4fa4-8ade-d9f1c652d5a6">
